src/ray/protobuf/serve.proto

```diff
@@ -131,6 +135,11 @@ message EndpointSet {
   map<string, EndpointInfo> endpoints = 1;
 }
 
+message ActorNameList {
+  repeated string names = 1;
+}
+
+// It is deprecated. Use ActorNameList instead.
 message ActorSet {
   repeated string names = 1;
 }
```

Comment:
can we just remove ActorSet now?

Reply:
Java LongPollClient depends on it now. I will replace it to ActorNameList in next Java API's PR.